packages/monosize-bundler-esbuild/package.json

```diff
@@ -2,6 +2,10 @@
   "name": "monosize-bundler-esbuild",
   "version": "0.1.5",
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/monosize"
+  },
```

Comment:
please also add 

`"directory": "<path to package>"`

Reply:
updated:)